doc/internal/index.md

```diff
@@ -0,0 +1,15 @@
+# Internal documentation of Node.js
+
+## CLI
+
+### Node.js options:
+
+#### `--inspect-brk-node[=[host:]port]`
+
+<!-- YAML
+added: v7.6.0
+-->
+
+Activate inspector on `host:port` and break at start of the first internal
+JavaScript script executed when the inspector is available.
```

Comment:
Double script sounds redundant to me. Perhaps, JavaScript code?

Reply:
`statement` is probably more accurate (it won't break on the first declaration).